pinot-controller/src/test/java/org/apache/pinot/controller/helix/ControllerTest.java

```diff
@@ -227,6 +227,8 @@ public Map<String, Object> getDefaultControllerConfiguration() {
     properties.put(ControllerConf.DISABLE_GROOVY, false);
     properties.put(ControllerConf.CONSOLE_SWAGGER_ENABLE, false);
     properties.put(CommonConstants.CONFIG_OF_TIMEZONE, "UTC");
+    // Disable resource util check in test so that each test can set the ResourceUtilizationInfo manually
+    properties.put(ControllerConf.RESOURCE_UTILIZATION_CHECKER_INITIAL_DELAY, 30_000);
```

Comment:
just thinking some more, instead of doing this for all tests, can you have your specific test implement the following:

```

  /**
   * Can be overridden to add more properties.
   */
  protected void overrideControllerConf(Map<String, Object> properties) {
  }
```

And have the above do this instead?